style.css

```diff
@@ -479,3 +513,339 @@ header button:hover {
   margin-top: 20px;
   margin-bottom: 8px;
 }
+
+@media screen and (min-width: 1024px) {
+  header {
+    height: 72px;
+  }
+
+  header .name {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  header .menu {
+    display: none;
+  }
+
+  header ul {
+    display: flex;
+    gap: 32px;
+    margin-left: auto;
+  }
+
+  .section-1 {
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 0;
+    gap: 40px;
+    background-image: url('./media/Illustration-header.svg');
+    background-position-y: 0;
+    background-position-x: 109%;
+  }
+
+  .section-1 .text-box {
+    order: 1;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+  }
+
+  .section-1 .social-media {
+    flex-direction: column;
+    width: auto;
+    height: auto;
+    margin: 0 36px;
+    gap: 10px;
+  }
+
+  .section-1 h1 {
+    width: 400px;
+    height: 160px;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .section-1 h2 {
+    width: 566px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .section-1 p {
+    width: 566px;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  #works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
+
+  #works h1 {
+    border: none;
+    color: #091e42;
+    width: auto;
+  }
+
+  #works .line {
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+    display: inline-block;
+    margin-left: 24px;
+  }
+
+  .card-1 {
+    flex-direction: row;
+    gap: 24px;
+    flex-shrink: auto;
+  }
+
+  #card-1-img {
+    width: auto;
+    height: auto;
+  }
+
+  #card-1-text {
+    align-self: flex-start;
+    gap: 5px;
+  }
+
+  .card-1 h2 {
+    margin-top: 5px;
+    margin-bottom: 19px;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .card-1 p {
+    width: 448px;
+    height: 140px;
+    font-size: 20px;
+    line-height: 28px;
+    margin: 0;
+  }
+
+  #projects-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    row-gap: 37px;
+    column-gap: 24px;
+  }
+
+  .repeated-card {
+    width: 369px;
+    height: 435px;
+    gap: 10px;
+    margin: 0;
+  }
+
```

Comment:
- [ ] I really appreciate your work, in fact you really have done a great job to make your design responsive while sticking to the design you chose and that's great👏. However, there is something you forgot to add that I think is very important for your portfolio. It is about your project cards. They look great but you forgot to add user interaction on them. See initially there should be **an overlay effect with texts describing the project** on each of your project cards(I mean those six aligned with `grid`), then when the user hovers on any of them, **the overlay effects and texts are supposed to disappear** and the user sees only the project image and the `see project` button.
**Your Design**
![image](https://user-images.githubusercontent.com/90222110/177874333-ac8d0cf0-693e-44e2-91d7-5e37657e63c7.png)
Looking at your design, you can see what the **first project card** looks like. **Each of your project cards is supposed to look like that when the user hovers on it** otherwise, there should be that overlay effect with the project description texts as it is on other project cards 🚀. These are important details that a developer should take into account because it is what makes him/her outstanding 😃. Please implement this feature so as to make your portfolio more professional. Let me hope that my explanations are really understandable 😺 .